src/models/commons.ts

```diff
@@ -12,11 +12,11 @@ export interface AssetType {
     * For example, for an asset class of ERC-20, the data holds the address of the token.
     * For ERC-721 data holds smart contract address and tokenId.
     **/
-    data: any,
+    data: any
 }
 
 export interface Asset {
-    amount: number,
+    value: number,
```

Comment:
I noticed you changed this. The docs in the contract say it is [an amount](https://github.com/rariblecom/protocol-contracts/blob/master/asset/contracts/LibAsset.md) which, I believe, is more informative than just using `value`. Do you disagree?

Reply:
If you read the language closely, the field is titled `value` I believe.  In their other API examples, they use `value`. [Here](https://docs.rarible.com/exchange/creating-a-sell-order) for example